runtime/src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java

```diff
@@ -466,7 +466,8 @@ public void stop(boolean shutdown) {
             CFUtils.within(cfBenchmarked, Duration.ofMillis(timeoutResponse));
         cfTimeout.exceptionally(e -> {
             outstandingRequests.remove(thisRequest);
-            log.debug("Remove request {} due to timeout!", thisRequest);
+            log.debug("Remove request {} to {} due to timeout! Message:{}",
```

Comment:
Probably it will more be performant not to call toString() on getNode(). Logger will do that when creating the log message only if the log level is met

Reply:
Done.